packages/cra-template-redux-typescript/template/src/features/counter/counterSlice.test.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  CounterSliceState,
+  counterSlice,
+  decrement,
+  increment,
+  incrementByAmount,
+} from "./counterSlice"
+
+describe("counter reducer", () => {
```

Comment:
possibly too close to an integration test, but i'd rather these use a real store and test selectors at the same time, e.g. 
```ts
let store = makeStore();
beforeEach(() => {
  store = makeStore();
})

it("should handle increment", () => {
  store = makeStore({ counter: { value: 3, status: "idle" } });
  expect(selectValue(store.getState())).toBe(3)
  store.dispatch(increment())
  expect(selectValue(store.getState())).toBe(4)
})
```